roles/build-ansible-test-images/defaults/main.yml

```
@@ -5,15 +5,20 @@ images_available:
   - name: localhost/test-image
     tag: archlinux
     script: ansible-test/archlinux/build.sh
-    python: "3.10"
+    pythons:
+      - "3.10"
   - name: localhost/test-image
     tag: centos-stream8
     script: ansible-test/centos-stream8/build.sh
-    python: "3.8"
+    pythons:
+      - "3.6"
+      - "3.8"
+      - "3.8"
```

```suggestion
      - "3.8"
      - "3.8"
```
should one of them be 3.9?

---

Indeed!
```suggestion
      - "3.8"
      - "3.9"
```